<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Laden</class>
 <widget class="QMainWindow" name="Laden">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>198</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Resonanz Fit - Messung einlesen</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>371</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticallayout">
     <item>
      <layout class="QHBoxLayout" name="horizontallayout2">
       <item>
        <widget class="QLineEdit" name="edit_pfad">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_aendern">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Ã„ndern...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer0">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="check_konfig">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Messkonfiguration aus Datei einlesen</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontallayout1">
       <item>
        <layout class="QGridLayout" name="gridlayout0">
         <item row="0" column="1">
          <widget class="QSpinBox" name="box_pixel">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_messpunkte">
           <property name="text">
            <string>Messpunkte pro Pixel</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="box_messpunkte">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_pixel">
           <property name="text">
            <string>Pixel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalspacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QGridLayout" name="gridlayout1">
         <item row="0" column="0">
          <widget class="QLabel" name="label_fmin">
           <property name="text">
            <string>Freq Min (Hz)</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="box_fmin">
           <property name="maximum">
            <number>999000</number>
           </property>
           <property name="singleStep">
            <number>500</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_fmax">
           <property name="text">
            <string>Freq Max (Hz)</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="box_fmax">
           <property name="minimum">
            <number>500</number>
           </property>
           <property name="maximum">
            <number>999000</number>
           </property>
           <property name="singleStep">
            <number>500</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer1">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontallayout0">
       <item>
        <widget class="QPushButton" name="button_fitten">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Fitten</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progress_bar">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
